src/components/BottomNav/BottomNav.style.ts

```diff
@@ -0,0 +1,52 @@
+import isPropValid from '@emotion/is-prop-valid'
+import styled from '@emotion/styled'
+import { Link, PathMatch } from 'react-router-dom'
+
+import { colors, sizes, typography, zIndex } from '@/shared/theme'
+import { animation } from '@/shared/theme/tokens'
+
+export const Container = styled.div`
+  width: 100%;
+  height: 64px;
+  background-color: ${colors.gray[800]};
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  z-index: ${zIndex.nearSheetOverlay};
+  display: flex;
+  transition: all ${animation.medium.timing} ${animation.medium.easing};
+
+  &.bottom-nav-active,
+  &.bottom-nav-exit {
+    bottom: 0;
+  }
+
+  &.bottom-nav-exit-active,
+  &.bottom-nav-enter {
+    bottom: -64px;
+  }
```

Comment:
Let's use `transform: translateY()` instead, it has much better transition performance than `bottom`